packages/hint-button-type/src/hint.ts

```diff
@@ -4,7 +4,7 @@
 
 import { HintContext } from 'hint/dist/src/lib/hint-context';
 import { IHint, ElementFound } from 'hint/dist/src/lib/types';
-import { debug as d } from 'hint/dist/src/lib/utils/debug';
+import { debug as d } from '@hint/utils/dist/src/debug';
```

Comment:
A few `/dist/src` that should be removed.

Reply:
This happens in quite a few files btw.

Reply:
I'm reviewing all the rules, but if the rule is used in the extension browser, we need to use the reference to the file.